x/ibc/07-tendermint/types/client_state_test.go

```diff
@@ -100,7 +95,7 @@ func (suite *TendermintTestSuite) TestVerifyClientConsensusState() {
 		// FIXME: uncomment
 		// {
 		// 	name:        "successful verification",
-		// 	clientState: ibctmtypes.NewClientState(chainID, chainID, height, commitmenttypes.GetSDKSpecs()),
+		// 	clientState: ibctmtypes.NewClientState(chainID, lite.chainID, height, commitmenttypes.GetSDKSpecs()),
```

Comment:
```suggestion
		// 	clientState: ibctmtypes.NewClientState(lite.DefaultTrustLevel, trustingPeriod, ubdPeriod, maxClockDrift, suite.header, commitmenttypes.GetSDKSpecs()),
```
I know its commented out, but seems more useful to have it be right